### ISSUE TITLE
IndexError when using proxy generator

### ISSUE THREAD

**Issue by gabrielepinto** [scholarly-python-package/scholarly#374]
**Describe the bug**
I am using scholarly 1.4.5.
I am trying to use the proxys, but when trying to load the proxy object to either "Single Proxy" or the ScraperApi proxy (i got a free api key), it gives me the following error

**To Reproduce**

```
from scholarly import scholarly, ProxyGenerator
pg = ProxyGenerator()
##lista_proxy[0]["http"] is the http address of my personal proxy

pg.SingleProxy(http = lista_proxy[0]["http"], https = lista_proxy[0]["https"])
scholarly.use_proxy(pg)
author = next(scholarly.search_author('Steven A Cholewiak'))
scholarly.pprint(author)`

```
error returned

```
IndexError                                Traceback (most recent call last)
<ipython-input-77-a1a102e2c38f> in <module>
      4 pg = ProxyGenerator()
      5 success = pg.SingleProxy(http = lista_proxy[0]["http"], https = lista_proxy[0]["https"])
----> 6 scholarly.use_proxy(pg)
      7 
      8 author = next(scholarly.search_author('Steven A Cholewiak'))

D:\anaconda\lib\site-packages\scholarly\_scholarly.py in use_proxy(self, proxy_generator, secondary_proxy_generator)
     63 
     64         """
---> 65         self.__nav.use_proxy(proxy_generator, secondary_proxy_generator)
     66 
     67 

D:\anaconda\lib\site-packages\scholarly\_navigator.py in use_proxy(self, pg1, pg2)
     84         else:
     85             self.pm2 = ProxyGenerator()
---> 86             proxy_works = self.pm2.FreeProxies()
     87             if not proxy_works:
     88                 self.logger.info("FreeProxy as a secondary proxy is not working. "

D:\anaconda\lib\site-packages\scholarly\_proxy_generator.py in FreeProxies(self, timeout)
    453         # at least once.
    454         for _ in range(60000):
--> 455             proxy = freeproxy.get()
    456             proxy_works = self._use_proxy(http=proxy, https=proxy)
    457             if proxy_works:

D:\anaconda\lib\site-packages\fp\fp.py in get(self)
     36 
     37     def get(self):
---> 38         proxy_list = self.get_proxy_list()
     39         if self.random:
     40             random.shuffle(proxy_list)

D:\anaconda\lib\site-packages\fp\fp.py in get_proxy_list(self)
     22             tr_elements = doc.xpath('//*[@id="proxylisttable"]//tr')
     23             if not self.country_id:
---> 24                 proxies = [f'{tr_elements[i][0].text_content()}:{tr_elements[i][1].text_content()}' for i in
     25                            range(1, 101)
     26                            if((tr_elements[i][4].text_content()) == 'anonymous' if self.anonym else True)]  # check the 5th column for `anonymous` if needed

D:\anaconda\lib\site-packages\fp\fp.py in <listcomp>(.0)
     22             tr_elements = doc.xpath('//*[@id="proxylisttable"]//tr')
     23             if not self.country_id:
---> 24                 proxies = [f'{tr_elements[i][0].text_content()}:{tr_elements[i][1].text_content()}' for i in
     25                            range(1, 101)
     26                            if((tr_elements[i][4].text_content()) == 'anonymous' if self.anonym else True)]  # check the 5th column for `anonymous` if needed

IndexError: list index out of range
```

**Desktop (please complete the following information):**
 - Proxy service: Single personal Proxy or ScraperAPI
 - python version: 3.8
 - OS: windows 10
 - Version 1.4.5.

**Do you plan on contributing?**
Your response below will clarify whether the maintainers can expect you to fix the bug you reported.
- [ ] Yes, I will create a Pull Request with the bugfix.

**Additional context**

**Comment by arunkannawadi** [comment]
This must mostly be a transient bug, meaning it should fix itself in a couple of hours. This is expected to become much more robust in v1.5 but for now, modifying your code to
```
scholarly.use_proxy(pg, pg)
```
should solve your issue.

**Comment by arunkannawadi** [comment]
Reading it again, this is not an actual error but a misleading traceback message. This will be fixed in v1.5. Closing as it is a duplicate of #313
